students/12th/heungsoolee/js/login.js

```diff
@@ -0,0 +1,41 @@
+const inputId = document.getElementsByClassName("id_input")[0];
+const inputPassword = document.getElementsByClassName("password_input")[0];
+const loginBtn = document.getElementsByClassName("login_button")[0];
+
+const slideImg = document.getElementsByClassName("slide_img");
+const slideClassName = "slide_now";
+
+const changeLoginBtn = () => {
+  let keyValue = event.key;
+
+  if (keyValue.length > 0) {
+    if (inputPassword.value.length > 0 && inputId.value.length > 0) {
+      loginBtn.style.backgroundColor = "#0095F6";
+      loginBtn.style.cursor = "pointer";
+    } else {
+      loginBtn.style.backgroundColor = "#C0DFFD";
+      loginBtn.style.cursor = "";
+    }
+  }
+};
+
+const slideShow = () => {
+  let nowImg = document.getElementsByClassName(slideClassName)[0];
+  if (nowImg) {
+    nowImg.classList.remove(slideClassName);
+    let nextImg = nowImg.nextElementSibling;
+
+    if (nextImg) {
+      nextImg.classList.add(slideClassName);
+    } else {
+      slideImg[0].classList.add(slideClassName);
+    }
+  } else {
+    slideImg[0].classList.add(slideClassName);
+  }
```

Comment:
```suggestion
  if (nowImg) {
    nowImg.classList.remove(slideClassName);
    let nextImg = nowImg.nextElementSibling;
    
    if (nextImg) return nextImg.classList.add(slideClassName);
    if (!nextImg) return slideImg[0].classList.add(slideClassName);
  }
  
  slideImg[0].classList.add(slideClassName);
```
- if-else 가 조금 정리 되면 좋을 것 같습니다. 중괄호의 깊이가 깊어지면 코드를 읽기 힘들어지기 때문입니다. 함수가 return문을 만났을 때 바로 종료된다는 점을 이용하면 위와 같이 정리할 수 있겠네요!